### DISCUSSION TITLE
Lifecycle status of the .NET Core Libraries built for .Net Framework 4.8

### DISCUSSION THREAD

**Discussion by segor** [dotnet/runtime#98314] [General] [answered]
Hello, 

My organization is currently auditing the security compliance of our .Net Framework 4.8 applications, and we have noticed that some .NET Core libraries have not been updated for many years. The list of libraries we use:
- `System.Buffers`, Nuget ver. [4.5.1](https://www.nuget.org/packages/System.Buffers/4.5.1)
- `System.Memory`, Nuget ver. [4.5.5](https://www.nuget.org/packages/System.Memory/4.5.5)
- `System.Numerics.Vectors`, Nuget ver. [4.5.0](https://www.nuget.org/packages/System.Numerics.Vectors/4.5.0)
- `System.Runtime.CompilerServices.Unsafe`, Nuget ver. [6.0.0](https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/6.0.0)

Given that .Net Framework 4.8 currently has no end date for support according to [this page](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework), we seek clarity on the lifecycle status of these libraries, whether they are part of the [dotnet/runtime](https://github.com/dotnet/runtime) repository and still supported, and if we can expect updates in case of critical issues. 

Thank you for your attention to this matter.

**Comment by huoyaoyuan** [comment, marked as answer]
They should still remain supported. 6.0 packages are built from 6.0 branch in this repo. Older packages are serviced at https://github.com/dotnet/maintenance-packages.

There were no updates recently since there are no new features, nor critical bugs or security risks.

@dotnet/area-infrastructure-libraries for more details.

**Reply by segor** [reply]
@huoyaoyuan Thank you for your answer. It is good to know that these libraries are supported and their source code is located at https://github.com/dotnet/maintenance-packages. I think it would be good to update these NuGet packages with Repository URL and Source Link. 
Also I cannot not find the sources of `System.Numerics.Vectors 4.5.0` which is quite popular - 343.4K downloads per day. Can you please help me with that?

**Reply by huoyaoyuan** [reply]
S.N.Vectors is still being migrated into that repo, you can see the active PR in that repo.

I don't know whether the link on nuget.org can be updated without publishing a new version.

**Reply by huoyaoyuan** [reply]
FYI: the official documentation about supporting policy https://github.com/dotnet/core/pull/9206

**Reply by carlossanlop** [reply]
> Also I cannot not find the sources of System.Numerics.Vectors

@segor you can now find it in dotnet/maintenance-packages as well: https://github.com/dotnet/maintenance-packages/tree/main/src/System.Numerics.Vectors

When you asked this question, I was in the process of migrating into that repo the source code of various packages that are still in support but whose source code was living inside branches that are already out of support. For reference, here's the progress status: https://github.com/dotnet/maintenance-packages/issues/42

**Reply by segor** [reply]
@carlossanlop @huoyaoyuan Thank you! I really appreciate your support.

**Comment by ViktorHofer** [comment]
cc @carlossanlop @ericstj
